src/network/networkprotocol.h

```diff
@@ -173,7 +178,12 @@ enum ToClientCommand
 		NOTE: The position in here is deprecated; position is
 		      explicitly sent afterwards
 	*/
-
+	TOCLIENT_ACCESS_DENIED = 0x0A,
+	/*
+		u16 command
```

Comment:
This information was useless with previous design because old design need to add opcode and then datas. With network packet the only useful thing is data